### DISCUSSION TITLE
Translate for PT-BR

### DISCUSSION THREAD

**Discussion by djotabr** [slyh/Cities2-TrafficLightsEnhancement#36] [Ideas]
Below is a translation into Brazilian Portuguese.
I don't know how to pull from Git, so follow my collaboration:

TrafficSignal = Sinal de Trânsito
Vanilla = Vanilla
SplitPhasing = Divisão de Fases
AdvancedSplitPhasing = Divisão Avançada de Fases
ProtectedRightTurns = Conversões à Direita Protegidas
ProtectedLeftTurns = Conversões à Esquerda Protegidas
Options = Opções
ExclusivePedestrianPhase = Fase Exclusiva para Pedestres
AlwaysGreenLeftTurns = Conversões à Esquerda Sempre Verdes
AlwaysGreenRightTurns = Conversões à Direita Sempre Verdes
LaneDirectionTool = Ferramenta de Direção de Pista (Experimental)
Close = Fechar
Open = Abrir
Reset = Redefinir
Save = Salvar
PleaseSave = Por favor, salve as alterações antes de selecionar outra junção
PleaseSelectJunction = Por favor, selecione uma junção
Left = À Esquerda
Ahead = À Frente
Right = À Direita
UTurn = Conversão em U

**Comment by slyh** [comment]
Added. Really grateful for your help!
